test/verify/check-machines

```diff
@@ -212,6 +212,8 @@ class TestMachines(MachineCase):
         b.wait_present("#test-vm-action-{0}".format(self.providerVm1_name)) # is provider-specific action button rendered?
         b.wait_in_text("#test-vm-props-{0}".format(self.providerVm1_name), "Test Provider Property") # are provider-specific VM props rendered?
 
+        b.click("{0}-sendNMI".format(self.providerVm1_id)) # click the Send Non-Maskable Interrupt button
```

Comment:
This test needs to be expanded to find out if the interrupt worked. Otherwise we aren't really testing anything.